app/src/main/java/com/duckduckgo/app/browser/BrowserTabFragment.kt

```diff
@@ -134,6 +138,21 @@ class BrowserTabFragment : Fragment(), FindListener {
         viewModel.onUserSubmittedQuery(text)
     }
 
+    override fun onStart() {
+        super.onStart()
+
+        omnibarTextInput.doOnLayout {
+            if (omnibarTextInput.hasFocus()) {
+                omnibarTextInput.postDelayed(300) { omnibarTextInput.showKeyboard() }
```

Comment:
Can we make it `omnibarTextInput?.showKeyboard` in case it has gone out of scope during the post delay period?

Also 300ms feels like a really long delay - I keep thinking there's a bug and then the keyboard pops up. Can we try 100ms here?